courses/management/commands/sync_courseruns.py

```diff
@@ -0,0 +1,86 @@
+"""
+Management command to sync grades and certificates for a course run
+"""
+from django.core.management.base import BaseCommand, CommandError
+
+from courses.models import CourseRun
+from courseware.api import get_edx_api_client
+from users.models import User
+from mitxpro.utils import now_in_utc
+
+
+class Command(BaseCommand):
+    """
+    Command to sync grades and certificates for a course run.
+    """
+
+    help = "Sync dates and title for all or a specific course run from edX."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--run",
+            type=str,
+            help="The 'courseware_id' value for a CourseRun",
+            required=False,
+        )
+        super().add_arguments(parser)
+
+    def handle(self, *args, **options):  # pylint: disable=too-many-locals
+        """Handle command execution"""
+        runs: [CourseRun] = []
+        if options["run"]:
+            try:
+                runs = [CourseRun.objects.get(courseware_id=options["run"])]
+            except CourseRun.DoesNotExist:
+                raise CommandError(
+                    "Could not find run with courseware_id={}".format(options["run"])
+                )
+        else:
+            now = now_in_utc()
+            runs = CourseRun.objects.filter(end_date__gt=now)
```

Comment:
are we just interested in `end_date__gt=now` runs yet only ? @pdpinch

Reply:
No. Imagine the scenario when a course ends in xPRO and the course team decides they need to add an extra week in Studio. 

If you must filter out some courses, it should only be the ones that have passed their archive date.

Reply:
@pdpinch archive date means expiration date, correct?

Reply:
That's right, expiration date.